app/components/DocumentTitle/index.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+
+const defaultTitle = 'Registratie';
+
+const DocumentTitle = ({ title }) => {
```

Comment:
Why not use [react helmet](https://github.com/nfl/react-helmet)?

Reply:
Overkill in my eyes. But can look into it a bit more. Also cause I've just realised that internationalisation should also be supported while changing titles.